diff --git a/gr-blocks/swig/blocks_swig1.i b/gr-blocks/swig/blocks_swig1.i
index 24483fc..7ea8054 100644
--- a/gr-blocks/swig/blocks_swig1.i
+++ b/gr-blocks/swig/blocks_swig1.i
@@ -42,24 +42,6 @@
 #include "gnuradio/blocks/tsb_vector_sink_i.h"
 #include "gnuradio/blocks/tsb_vector_sink_s.h"
 #include "gnuradio/blocks/throttle.h"
-#include "gnuradio/blocks/vector_map.h"
-#include "gnuradio/blocks/vector_to_stream.h"
-#include "gnuradio/blocks/vector_to_streams.h"
-#include "gnuradio/blocks/vector_insert_b.h"
-#include "gnuradio/blocks/vector_insert_s.h"
-#include "gnuradio/blocks/vector_insert_i.h"
-#include "gnuradio/blocks/vector_insert_f.h"
-#include "gnuradio/blocks/vector_insert_c.h"
-#include "gnuradio/blocks/vector_sink_b.h"
-#include "gnuradio/blocks/vector_sink_s.h"
-#include "gnuradio/blocks/vector_sink_i.h"
-#include "gnuradio/blocks/vector_sink_f.h"
-#include "gnuradio/blocks/vector_sink_c.h"
-#include "gnuradio/blocks/vector_source_b.h"
-#include "gnuradio/blocks/vector_source_s.h"
-#include "gnuradio/blocks/vector_source_i.h"
-#include "gnuradio/blocks/vector_source_f.h"
-#include "gnuradio/blocks/vector_source_c.h"
 #include "gnuradio/blocks/wavfile_sink.h"
 #include "gnuradio/blocks/wavfile_source.h"
 %}
@@ -79,24 +61,6 @@
 %include "gnuradio/blocks/tsb_vector_sink_i.h"
 %include "gnuradio/blocks/tsb_vector_sink_s.h"
 %include "gnuradio/blocks/throttle.h"
-%include "gnuradio/blocks/vector_map.h"
-%include "gnuradio/blocks/vector_to_stream.h"
-%include "gnuradio/blocks/vector_to_streams.h"
-%include "gnuradio/blocks/vector_insert_b.h"
-%include "gnuradio/blocks/vector_insert_s.h"
-%include "gnuradio/blocks/vector_insert_i.h"
-%include "gnuradio/blocks/vector_insert_f.h"
-%include "gnuradio/blocks/vector_insert_c.h"
-%include "gnuradio/blocks/vector_sink_b.h"
-%include "gnuradio/blocks/vector_sink_s.h"
-%include "gnuradio/blocks/vector_sink_i.h"
-%include "gnuradio/blocks/vector_sink_f.h"
-%include "gnuradio/blocks/vector_sink_c.h"
-%include "gnuradio/blocks/vector_source_b.h"
-%include "gnuradio/blocks/vector_source_s.h"
-%include "gnuradio/blocks/vector_source_i.h"
-%include "gnuradio/blocks/vector_source_f.h"
-%include "gnuradio/blocks/vector_source_c.h"
 %include "gnuradio/blocks/wavfile_sink.h"
 %include "gnuradio/blocks/wavfile_source.h"
 
@@ -115,23 +79,6 @@ GR_SWIG_BLOCK_MAGIC2(blocks, tsb_vector_sink_f);
 GR_SWIG_BLOCK_MAGIC2(blocks, tsb_vector_sink_i);
 GR_SWIG_BLOCK_MAGIC2(blocks, tsb_vector_sink_s);
 GR_SWIG_BLOCK_MAGIC2(blocks, throttle);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_map);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_to_stream);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_to_streams);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_insert_b);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_insert_s);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_insert_i);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_insert_f);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_insert_c);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_sink_b);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_sink_s);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_sink_i);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_sink_f);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_sink_c);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_source_b);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_source_s);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_source_i);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_source_f);
-GR_SWIG_BLOCK_MAGIC2(blocks, vector_source_c);
+
 GR_SWIG_BLOCK_MAGIC2(blocks, wavfile_sink);
 GR_SWIG_BLOCK_MAGIC2(blocks, wavfile_source);
