--- volk/CMakeLists.orig.txt	2015-06-20 08:11:37.413183163 +0200
+++ volk/CMakeLists.txt	2015-06-20 08:12:47.512787331 +0200
@@ -106,19 +106,19 @@
 ########################################################################
 # Setup doxygen
 ########################################################################
-find_package(Doxygen)
-if(DOXYGEN_FOUND)
-    configure_file(
-        ${CMAKE_SOURCE_DIR}/Doxyfile.in
-        ${CMAKE_BINARY_DIR}/Doxyfile
-    @ONLY)
-
-    add_custom_target(doc
-        ${DOXYGEN_EXECUTABLE} ${CMAKE_BINARY_DIR}/Doxyfile
-        WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-        COMMENT "Generating documentation with Doxygen" VERBATIM
-    )
-endif(DOXYGEN_FOUND)
+#find_package(Doxygen)
+#if(DOXYGEN_FOUND)
+#    configure_file(
+#        ${CMAKE_SOURCE_DIR}/Doxyfile.in
+#        ${CMAKE_BINARY_DIR}/Doxyfile
+#    @ONLY)
+#
+#    add_custom_target(doc
+#        ${DOXYGEN_EXECUTABLE} ${CMAKE_BINARY_DIR}/Doxyfile
+#        WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
+#        COMMENT "Generating documentation with Doxygen" VERBATIM
+#    )
+#endif(DOXYGEN_FOUND)
 
 ########################################################################
 # Setup the package config file
@@ -249,8 +249,8 @@
 ########################################################################
 # And the utility apps
 ########################################################################
-add_subdirectory(apps)
-add_subdirectory(python/volk_modtool)
+#add_subdirectory(apps)
+#add_subdirectory(python/volk_modtool)
 
 ########################################################################
 # Print summary
--- volk/lib/CMakeLists.orig.txt	2015-07-06 14:31:43.314986949 +0200
+++ volk/lib/CMakeLists.txt	2015-07-06 14:29:39.260518695 +0200
@@ -538,7 +538,7 @@
 
     #Configure target properties
     set_target_properties(volk_obj PROPERTIES COMPILE_FLAGS "-fPIC")
-    set_target_properties(volk PROPERTIES SOVERSION ${LIBVER})
+#    set_target_properties(volk PROPERTIES SOVERSION ${LIBVER})
     set_target_properties(volk PROPERTIES DEFINE_SYMBOL "volk_EXPORTS")
 
     #Install locations
@@ -564,7 +564,7 @@
     #create the volk runtime library
     add_library(volk SHARED ${volk_sources})
     target_link_libraries(volk ${volk_libraries})
-    set_target_properties(volk PROPERTIES SOVERSION ${LIBVER})
+#    set_target_properties(volk PROPERTIES SOVERSION ${LIBVER})
     set_target_properties(volk PROPERTIES DEFINE_SYMBOL "volk_EXPORTS")
 
     install(TARGETS volk
